app/config/AppConfig.scala

```diff
@@ -111,3 +112,8 @@ case class TrackingConsentFrontend(gtm: Gtm, url: String)
 case class Platform(frontend: Frontend)
 
 case class Frontend(host: Option[String])
+
+case class SecureMessaging(protocol: Option[String], host: String, port: Option[Int], fetchInbox: String) {
+  def fetchInboxEndpoint(): String =
```

Comment:
Sorry, Tim :upside_down_face: ... maybe worth also set this as `lazy val` rather than `def` ?

Reply:
Ok, changed